widget/playground/src/containers/configContainer/ConfigContainer.tsx

```diff
@@ -118,6 +89,22 @@ export function ConfigContainer(props: PropsWithChildren) {
             color="neutral700">
             To use the Playground page, You must use the Desktop version
           </Typography>
+          <StyledButton
+            type="secondary"
+            size="medium"
+            variant="contained"
+            onClick={() => window.open(RANGO_DOCS_URL, '_blank')}
+            css={{ marginTop: '$40' }}>
```

Comment:
You could probably use divider instead of margin here.

Reply:
fixed